mne/tests/test_import_nesting.py

```diff
@@ -26,7 +26,8 @@
 # check sklearn and others
 for x in sys.modules.keys():
     for key in ('sklearn', 'pandas', 'pyvista', 'matplotlib',
-                'dipy', 'nibabel', 'cupy', 'picard', 'pyvistaqt', 'pooch'):
+                'dipy', 'nibabel', 'cupy', 'picard', 'pyvistaqt', 'pooch',
+                'tqdm', 'jinja2'):
```

Comment:
Do we still want to test required packages?

Reply:
Yes. We should nest as many as possible/reasonable to reduce import time. `pooch` and `jinja2` aren't used all that often so we might as well nest them. (`decorator`, on the other hand, is core to `verbose` and `fill_doc` so we have to import it at the root level.)